tests/sync/courseDb.test.js

```diff
@@ -213,8 +213,6 @@ describe('course database', () => {
         await writeQuestion(dir, 'question2', getQuestion());
         await writeQuestion(dir, 'question3', getAlternativeQuestion());
         const result = await courseDb.loadQuestions(dir);
-        assert.isFalse(infofile.hasErrors(result));
-        assert.isFalse(infofile.hasWarnings(result));
```

Comment:
Don't we still want zero warnings/errors here?

Reply:
This was just straight up invalid code that type-checking caught once I added required properties to infofiles. `result` isn't an infofile-like thing that we can pass to `hasErrors` or `hasWarnings`. We check for errors/warnings a few lines down on individual objects within `result`.

Reply:
Ha, nice 😄